### ISSUE TITLE
error_message() does not work

### ISSUE THREAD

**Issue by jaygambetta** [Qiskit/qiskit-terra#1504]
running 

```
job = backend.retrieve_job(ran_job.job_id())
```

on a previous job that had an error than doing 

```
job.error_message() 
```

returns no message

**Comment by ajavadia** [comment]
This must be fixed in the IBMQConnector module.

**Comment by jaygambetta** [comment]
I would like this fixed in 0.8. @diego-plan9 can i assign this to you.

**Comment by diego-plan9** [comment]
Yes - I think somebody else inspected the root cause when the issue was created and might be able to provide more info: maybe @delapuente ?
